src/codec_aom.c

```diff
@@ -528,11 +535,22 @@ static avifResult aomCodecEncodeImage(avifCodec * codec,
 #endif
         int aomCpuUsed = -1;
         if (encoder->speed != AVIF_SPEED_DEFAULT) {
-            if (encoder->speed < 8) {
-                aomCpuUsed = AVIF_CLAMP(encoder->speed, 0, 6);
-            } else {
+            aomCpuUsed = AVIF_CLAMP(encoder->speed, 0, 9);
+            if (aomCpuUsed >= 7) {
+#if defined(AOM_USAGE_ALL_INTRA) && defined(ALL_INTRA_HAS_SPEEDS_7_TO_9)
+                // Speeds 7-9 in all intra mode work.
+                // https://aomedia-review.googlesource.com/c/aom/+/140624
+                // If you were using libaom with lower encoder ABI version
+                // and using the following avif speed setting:
+                //  - speed 0-6: no change is needed
+                //  - speed 7:   change to speed 6 for the same results
+                //  - speed 8-9: re-test and re-adjust speed according to your app needs
```

Comment:
Please delete this block comment (lines 541-547).

Lines 541-542 are now available at line 46.

Lines 543-547 only need to be in the release notes/CHANGELOG.md.

Reply:
Done